clearnode/tag_test.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Why using `assert` in the first place? It's generally a good idea for a test case to fail as early as possible